pkg/granted/cf.go

```diff
@@ -0,0 +1,89 @@
+package granted
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/common-fate/clio"
+	"github.com/common-fate/granted/pkg/cfaws"
+	"github.com/common-fate/granted/pkg/cfcfg"
+	sdkconfig "github.com/common-fate/sdk/config"
+	"github.com/pkg/browser"
+	"github.com/urfave/cli/v2"
+)
+
+var CFCommand = cli.Command{
+	Name:  "cf",
+	Usage: "Open Common Fate in the browser",
+	Flags: []cli.Flag{&cli.StringFlag{Name: "profile", Usage: "open the console for a specific profile"}},
+	Action: func(c *cli.Context) error {
+
+		ctx := c.Context
+		consoleURL := ""
+		profiles, err := cfaws.LoadProfiles()
+		if err != nil {
+			return err
+		}
+
+		profileName := c.String("profile")
+		if profileName != "" {
+			p, err := profiles.Profile(profileName)
+			if err != nil {
+				return err
+			}
+			url, err := cfcfg.GetCommonFateURL(p)
+			if err != nil {
+				return err
+			}
+			if url == nil {
+				return errors.New("the profile exists but it is not configured with with a Common Fate console url")
+			}
+			consoleURL = url.String()
```

Comment:
If `--profile` flag is provided, what is the UX here? It looks like it will continue down and still ask which deployment to open. Should this be returning early here?

```
consoleURL = url.String()
return browser.OpenURL(consoleURL)
```

Reply:
Fixed this now